R-graphql/example.r

```diff
@@ -3,8 +3,8 @@ library("jsonlite")
 library("httr")
 
 API_URL <- "https://api.santiment.net/graphql"
-# change with real Apikey
-API_KEY <- Sys.getenv("SANBASE_API_KEY")
+# change with real Apikey in the .Renviron file
```

Comment:
Also one can execute like this: `SANBASE_API_KEY="Enter personal API key" Rscript example.r`